nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -24,6 +24,9 @@ let
         # Name appended to menuentry defaults to params if no specific name given.
         option.name or (optionalString (option ? params) "(${option.params})")
         }' ${optionalString (option ? class) " --class ${option.class}"} {
+          # Fallback to UEFI console for boot, efifb sometimes has difficulties.
+          terminal_output console
```

Comment:
For context, the stage-1 and up until the modesetting drived kicked in, the output was severely glitched out on the Steam Deck.

Anyway there are *literally no reasons* not to do that, as at that point we're already pivoting into the Linux kernel.